app/Functions/Functions.php

```diff
@@ -1809,6 +1870,7 @@ public static function getRelationshipNameFromPath($path, Individual $person1 =
 					//
 					// Need to find out which languages use which rules.
 					switch (WT_LOCALE) {
+						case 'de':
```

Comment:
Proper number for great-x-uncle/aunt. Translations will have to be adjusted once this is accepted.